src/libmongoc/src/mongoc/mongoc-topology.c

```diff
@@ -383,6 +383,14 @@ mongoc_topology_new (const mongoc_uri_t *uri, bool single_threaded)
 #endif
 
    topology = (mongoc_topology_t *) bson_malloc0 (sizeof *topology);
+   // Check if requested to use TCP for SRV lookup.
+   {
+      char *MONGOC_EXPERIMENTAL_SRV_PREFER_TCP = _mongoc_getenv ("MONGOC_EXPERIMENTAL_SRV_PREFER_TCP");
```

Comment:
`char *MONGOC_EXPERIMENTAL_SRV_PREFER_TCP` -> `char *srv_prefer_tcp` (not a macro).